Resources/Prototypes/Entities/Structures/Wallmounts/switch.yml

```diff
@@ -509,6 +509,333 @@
   - type: AccessReader
     access: [["Atmospherics"]]
 
+# big red button
+- type: entity
+  id: BigRedButton
```

Comment:
why doesnt this inherit from signal switch

Reply:
because it doesn't go on the wall, it goes on a table

Reply:
it will still work even if there is no wall under it, and you could still change the 1 or 2 components if not

Reply:
yeah alright